src/modules/UI/scenes/TransactionList/reducer.js

```diff
@@ -1,13 +1,23 @@
 // @flow
 
 import type { EdgeTransaction } from 'edge-core-js'
-import { combineReducers } from 'redux'
+import { type Reducer, combineReducers } from 'redux'
 
+import type { GuiContact, TransactionListTx } from '../../../../types.js'
 import type { Action } from '../../../ReduxTypes.js'
 
-export type TransactionsState = Array<EdgeTransaction>
+export type TransactionListState = {
+  +contactsList: Array<GuiContact>, // MISSING!!!
+  +currentCurrencyCode: string,
+  +currentEndIndex: number,
+  +currentWalletId: string,
+  +numTransactions: number,
+  +searchVisible: boolean,
+  +transactions: Array<TransactionListTx>, // Array<EdgeTransaction> !?
```

Comment:
The reducer is returning a different type than Redux is expecting.

Reply:
`TransactionListTx` is correct since the GUI modifies the arrays